.cicd/pipeline.yml

```diff
@@ -158,6 +169,17 @@ steps:
       - "queue=mac-anka-node-fleet"
     skip: ${SKIP_MACOS_10_15}${SKIP_MAC}${SKIP_WASM_METRICS}
 
+  - label: ":oracle_linux: Oracle_Linux 8 - EOSIO.CONTRACTS Build"
+    command:
+      - "buildkite-agent artifact download build.tar.gz . --step ':oracle_linux: Oracle_Linux 8 - Build' --agent-access-token $$BUILDKITE_AGENT_ACCESS_TOKEN && tar -xzf build.tar.gz"
+      - "./.cicd/eosio-contracts-build.sh"
+    env:
+      IMAGE_TAG: "oraclelinux-8"
+    agents:
+      queue: "automation-eks-eos-tester-fleet"
+    timeout: ${TIMEOUT:-10}
+    skip: ${SKIP_ORACLE_LINUX_8}${SKIP_LINUX}${SKIP_WASM_METRICS}
+
```

Comment:
This is missing a key definition. See the other `EOSIO.CONTRACTS Build` sections.